data/en/imageformats.json

```diff
@@ -4,11 +4,19 @@
 	"syntax": "imageFormats()",
 	"returns": "struct",
 	"related": [],
-	"description": "",
+	"description": "Returns all supported image formats that can be read and written.",
 	"params": [],
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/imageformats.html"}
 	},
-	"examples": [],
+	"examples": [
+    {
+      "title": "Simple example",
+      "description": "Dumps the structure returned by the function",
```

Comment:
Could you explain what happens here better?

Reply:
I've improved the description a in https://github.com/foundeo/cfdocs/pull/870/commits/0e8ea960f31d382c553cff4df1cbd8e1c010140d. Hope that makes it a bit clearer what the example does.